lib/capybara/webkit/matchers.rb

```diff
@@ -7,11 +7,11 @@ module RspecMatchers
           actual.error_messages.any?
         end
 
-        failure_message_for_should do |actual|
+        failure_message do |actual|
```

Comment:
Hey @gabebw. I think `alias_method` would be just fine. That's the pattern I've seen elsewhere.